src/components/Categories/LoanSpotlight.vue

```diff
@@ -0,0 +1,255 @@
+		KvGrid,
+		KvButton,
+		KvPageContainer,
+		KvResponsiveImage
+	},
+	inject: ['apollo', 'cookieStore'],
+	data() {
+		return {
+			spotlightPlaceholderImageCTF: '',
+			spotlightLoan: {},
+			targetedLoanChannelID: null
+		};
+	},
+	computed: {
+		altText() {
+			return "Today's loan spotlight";
+		},
+		getSpotlightText() {
+			return toParagraphs(this.spotlightLoan.description);
+		},
+		getSpotlightLoanLocation() {
+			if (this.spotlightLoan.geocode.city && this.spotlightLoan.geocode.country.name) {
+				return `${this.spotlightLoan.geocode.city}, ${this.spotlightLoan.geocode.country.name}`;
+			}
+			return `${this.spotlightLoan.geocode?.country?.name ?? ''}`;
+		},
+		imageExists() {
+			return this.spotlightLoan.image?.default ?? '';
+		},
+		getSpotlightImage() {
+			if (this.imageExists === '') {
+				return [['small', this.spotlightPlaceholderImageCTF]];
+			}
+			return [['small', this.spotlightLoan.image.retina], ['small retina', this.spotlightLoan.image.retina]];
+		}
+	},
+	apollo: {
+		preFetch(config, client) {
+			return client.query({
+				query: allChannelsQuery
+			}).then(({ data }) => {
+				const targetedLoanChannelURL = routePath;
+				const targetedLoanChannelID = getTargetedChannel(targetedLoanChannelURL, data);
+				return client.query({
+					query: spotlightLoanQuery,
+					variables: {
+						ids: [targetedLoanChannelID],
+					},
+				});
+			});
+		},
+	},
+	created() {
+		let allChannelsData = {};
+		try {
+			allChannelsData = this.apollo.readQuery({
+				query: allChannelsQuery,
+			});
+		} catch (e) {
+			logReadQueryError(e);
+		}
+
+		const targetedLoanChannelURL = routePath;
+		// isolate recommended-by-lenders channel id
+		this.targetedLoanChannelID = getTargetedChannel(targetedLoanChannelURL, allChannelsData);
+
+		// Read the spotlight data from the cache
+		let spotlightData = {};
+		try {
+			spotlightData = this.apollo.readQuery({
+				query: spotlightLoanQuery,
+				variables: {
+					ids: [this.targetedLoanChannelID],
+				},
+			});
+		} catch (e) {
+			logReadQueryError(e);
+		}
+		// filter out loans with anonymizationLevel of full, then take first in list
+		this.spotlightLoan = filterByAnonymizationLevel(spotlightData);
+	},
+	mounted() {
+		this.apollo.query({
+			query: gql`
+				query bpHeroBackgroundImage($placeholderKey: String) {
+					contentful {
+						placeholder: entries(contentType: "background", contentKey: $placeholderKey)
+					}
+				}
+			`,
+			variables: {
+				placeholderKey: 'bp-hero-country-placeholder',
+			},
+		}).then(result => {
+			const placeholderMedia = result?.data?.contentful?.placeholder?.items?.[0]?.fields?.backgroundMedia ?? {};
+			this.spotlightPlaceholderImageCTF = placeholderMedia?.fields?.file?.url ?? '';
+		});
+	},
+};
+
+</script>
+
+<style lang="postcss" scoped>
+
+.spotlight-loan-image >>> img {
+	@apply tw-object-cover tw-object-top tw-rounded;
+	@apply tw-w-full tw-h-full md:tw-w-[390px] md:tw-h-[320px] lg:tw-w-[520px] lg:tw-h-[390px];
```

Comment:
These widths and heights seem oddly specific, for this component to work in different contexts we might not need to be so specific. What happens if we do something like: `@apply tw-w-full tw-h-full md:tw-w-1/2 md:tw-max-h-[320px] lg:tw-max-h-[390px]` ? Setting width to be 50% and height to be a max-height instead?

Reply:
Yeah they were pretty specific haha. The large size dimensions were what Nathan had, but the medium size dimensions were meant to avoid wrapping of the title "Today's loan spotlight".

Reply:
Just as a reference, this is how it looks with the original dimensions I had.

<img width="913" alt="Screen Shot 2022-07-08 at 11 45 00 AM" src="https://user-images.githubusercontent.com/90521087/178026154-60f1e355-1960-4c66-9341-436b29f89a13.png">

Reply:
I did some quick testing locally and the spotlight image was really tall in small view. Do we want a max height? Otherwise looks good!

![image](https://user-images.githubusercontent.com/16867161/178029930-4e581b5a-0ac7-4feb-89fb-88b5df3f4a3a.png)

Reply:
> @apply tw-w-full tw-h-full md:tw-w-[1/2] md:tw-max-h-[320px] lg:tw-max-h-[390px];

I actually like this solution with maybe adding a max width as well? Lets create 2 stories one for vertical and one with horizontal loan image and we can run this by Nathan